### PR TITLE
Update PR template

### PR DESCRIPTION
**Pull request by c0c0n3**
## Proposed changes

* Add running tests locally to PR template checklist.
* Add linting locally to PR template checklist.
* Reformat text to work around GitHub broken markdown renderer---newlines break sentences.

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

- [x] I have read the [CONTRIBUTING][contrib] doc
- [x] I have signed the [CLA][cla]
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have updated the [RELEASE NOTES][release]
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

## Further comments

N/A

[cla]: https://raw.githubusercontent.com/orchestracities/ngsi-timeseries-api/master/individual_cla.pdf
    "Martel Open Source Software Individual Contributor License Agreement"
[contrib]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/CONTRIBUTING.md
    "Contributing to QuantumLeap"
[release]: https://github.com/orchestracities/ngsi-timeseries-api/blob/master/RELEASE_NOTES.md
    "QuantumLeap Release Notes"

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,35 +7,29 @@ that issue.
...
-to look for before merging your code._
+_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
 
 - [ ] I have read the [CONTRIBUTING][contrib] doc
 - [ ] I have signed the [CLA][cla]
-- [ ] I have added tests that prove my fix is effective or that my
-      feature works
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] I have run **all** the existing tests locally (not just those related to my feature) and there are no errors
+- [ ] After the last push to the PR branch, I have run the lint script locally and there are no changes to the code base
```
Comment (chicco785) [inline .github/pull_request_template.md:25]: i thought these lines were more than 80 chars, but markdown linter is not complaining.... so good to go

**Review by chicco785 — APPROVED** [review]

File: .github/pull_request_template.md (hunk 1 above)
Comment (chicco785) [reply .github/pull_request_template.md:25]: mmm it maybe that being a "." directory is kind of hidden

Comment (c0c0n3) [reply .github/pull_request_template.md:25]: the problem is that the PR template renderer is broken. If you put this markdown anywhere in GH

```
This is a long sentence
I'd like to break
```

It gets rendered as

> This is a long sentence I'd like to break

except when it comes to the PR template where it'd be rendered as

> This is a long sentence
> I'd like to break

Comment (c0c0n3) [reply .github/pull_request_template.md:25]: That's the reason why I reformatted the file to keep paragraphs on one line, even if they go over 80 chars. It sucks and we can reformat again if the GH guys fix the renderer.

**Pull request merged**